### PR TITLE
Update Serbian translation

### PR DESCRIPTION
**Pull request by DigitLib**
Added also translation for Collection dialog for 'collection' and button 'close'.

### CODE REVIEW

File: src/locales/sr/translation.json [hunk 1]
```diff
@@ -126,6 +131,10 @@
     "thumbnailNavigation": "Сличице",
     "thumbnails": "Сличице",
     "toggleWindowSideBar": "Сајдбар",
+    "collection": "Колекција",
+    "totalCollections": "{{count}} колекција/е",
+    "totalManifests": "{{count}} објек(а)та",
+    "close": "Затворите",
```
Comment (mejackreed) [inline src/locales/sr/translation.json:137]: Is this `close` key needed here?

File: src/locales/sr/translation.json [hunk 2]
```diff
@@ -126,6 +131,10 @@
     "thumbnailNavigation": "Сличице",
     "thumbnails": "Сличице",
     "toggleWindowSideBar": "Сајдбар",
+    "collection": "Колекција",
```
Comment (mejackreed) [inline src/locales/sr/translation.json:134]: Is this `collection` key needed here?

**Review by mejackreed — CHANGES_REQUESTED** [review]
Looks great thank you for the contribution! I had one quick question about the usage of two of the keys added.

File: src/locales/sr/translation.json (hunk 1 above)
Comment (DigitLib) [reply src/locales/sr/translation.json:137]: Yes, it is on dialog panel, i suppose that this place is correct. Should it be in abecede direction?

File: src/locales/sr/translation.json (hunk 2 above)
Comment (DigitLib) [reply src/locales/sr/translation.json:134]: Also in abecede?

File: src/locales/sr/translation.json (hunk 1 above)
Comment (mejackreed) [reply src/locales/sr/translation.json:137]: Thank you, yes please add in alphabetical direction!

File: src/locales/sr/translation.json (hunk 2 above)
Comment (mejackreed) [reply src/locales/sr/translation.json:134]: Thank you, yes please add in alphabetical direction!

**Comment by DigitLib** [comment]
done

**Review by mejackreed — APPROVED** [review]

**Pull request merged**